### PR TITLE
feat (angular-dialog) Log errors via angular dialog 

### PR DESCRIPTION
**Pull request by jafin**
Relates to #8 Removed the window.confirm and replaced with native angular dialog confirm.
Remove Angular-Flash use the AngularUI messages.
Remove old jquery 2.1.0   (project is referencing 2.1.4)
Swaped the dialog in Home with the AngularIsh version
Swaped jquery calls in home with angular $http calls.

### CODE REVIEW

File: src/SbManager/SbManager.csproj [hunk 1]
```diff
@@ -302,24 +307,30 @@
...
-    <Content Include="Scripts\jquery-1.9.1.intellisense.js" />
-    <Content Include="Scripts\jquery-2.1.0.js">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Scripts\jquery-2.1.0.min.js">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    <Content Include="Scripts\dialogs.min.js">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
+    <Content Include="Scripts\jquery-1.9.1.intellisense.js" />
```
Comment (drewfreyling) [inline src/SbManager/SbManager.csproj]: Does this file still need to be here?

File: src/SbManager/Scripts/manager/filters.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+﻿///Usage <any nguid-bind-html="value | unsafe"></any>
+$app.filter('unsafe', function ($sce) { return $sce.trustAsHtml; });
```
Comment (drewfreyling) [inline src/SbManager/Scripts/manager/filters.js]: I'd recommend placing this in an IIFE

File: src/SbManager/Scripts/manager/homeController.js [hunk 3]
```diff
@@ -1,18 +1,34 @@
...
+
+    $scope.alerts = alertService.get();
     $scope.refresh();
 
     $scope.deleteAll = function () {
-        if (!window.confirm("You sure? This can't be undone and your world might explode.")) return;
-        $scope.model = null;
-        $.post(window.applicationBasePath + "/api/v1/busmanager/deleteall", function (d) {
-            window.location = "#/";
+        var dlg = dialogs.confirm("Delete everything?", "You are about to delete all the queues and topics on this bus. Whats done cannot be undone.");
```
Comment (drewfreyling) [inline src/SbManager/Scripts/manager/homeController.js]: Minor: "Whats" should be "What is"

File: src/SbManager/Views/master.html [hunk 4]
```diff
@@ -4,16 +4,23 @@
...
-    <link rel="stylesheet" href="/Content/bootstrap.superhero.min.css" />
+    <link rel="stylesheet" href="/Content/bootstrap.superhero.min.css"/>
     <link rel="stylesheet" href="/Content/site.css" />
-    <link rel="stylesheet" href="/Content/font/sbmanagerfont.css" />
+    <link rel="stylesheet" href="/Content/font/sbmanagerfont.css"/>
+    <link rel="stylesheet" href="/Content/dialogs.min.css" />
+    
     <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.2.28/angular.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.2.28/angular-route.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.2.28/angular-route.js"></script>
```
Comment (drewfreyling) [inline src/SbManager/Views/master.html]: This should be the minified version.

File: src/SbManager/Views/index.html [hunk 5]
```diff
@@ -2,4 +2,6 @@
 
 @Section['Content']
 <div data-ng-view=""></div>
+
+<div flash-message="10000"></div>
```
Comment (drewfreyling) [inline src/SbManager/Views/index.html]: I thought you were removing angular-flash?

Comment (jafin) [inline src/SbManager/Views/index.html]: Yeah missed that, was removed on the 1.4 branch i'm on now, just edit it out.

**Comment by jafin** [comment]
@drewfreyling,  comments on prior PR have been actioned.

**Comment by drewfreyling** [comment]
@jafin nice work. Unfortunately there are no conflicts as I just merged your other PR!

**Comment by drewfreyling** [comment]
@jafin any progress on the merge conflicts?

**Comment by drewfreyling** [comment]
@jafin if this isn't going to be worked on I may have to close it.

**Comment by jafin** [comment]
Close it, I doubt I'll have any changes in the short term.

**Pull request closed without merging**